packages/semcom-demo-pods/data/metadata/.acl

```diff
@@ -0,0 +1,20 @@
+@prefix  acl:   <http://www.w3.org/ns/auth/acl#>.
+@prefix  foaf:  <http://xmlns.com/foaf/0.1/>.
+
+<#tony>
+    a               acl:Authorization;
+    acl:agent       <https://pods.actueel-inkomen.digita.ai/tiapieters/profile/card#me>;
```

Comment:
This obviously isn't correct... must be a localhost WebID, and one of Tony instead of Tia.

Reply:
If other WebIDs in the data are not local, they should also be made local.